libraries/Microsoft.Bot.Schema/SuggestedActions.cs

```diff
@@ -35,10 +35,10 @@ public SuggestedActions()
         /// shown to.  These Ids are relative to the channelId and a subset of
         /// all recipients of the activity</param>
         /// <param name="actions">Actions that can be shown to the user</param>
-        public SuggestedActions(IList<string> to = default(IList<string>), IList<CardAction> actions = default(IList<CardAction>))
+        public SuggestedActions(IEnumerable<string> to = default(IEnumerable<string>), IEnumerable<CardAction> actions = default(IEnumerable<CardAction>))
         {
-            To = to;
-            Actions = actions;
+            To = to.ToList();
```

Comment:
These can be `null`, so the `.ToList()` call would fail with a `NullReferenceException`. I believe you should be able to remedy this easily with a null propagating operator:

```CSharp
To = to?.ToList();
```

Reply:
I just noticed this actually broke one of the tests in the build, so we have some coverage of this scenario already at least.

Reply:
Good catch. Just updated the PR.